packages/boxel/addon/components/boxel/input/ranged-number-picker/index.ts

```diff
@@ -0,0 +1,32 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import './index.css';
+
+interface RangedBoxelInputRangedNumberPickerInputArgs {
+  min: number;
+  max: number;
+  onNumberSelected: (num: number) => void;
+}
```

Comment:
This is an older style of component interface. We should use the newer style that Glint uses.